api/_lib/script/pack.ts

```diff
@@ -46,6 +37,20 @@ async function main() {
   }
 }
 
+export function modifyPackageJson(task, packageObj) {
+  for (const dependency of task.dependencies) {
+    const name = dependency.split('#')[0];
+    const tarballUrl = `https://${process.env.VERCEL_URL}/tarballs/${name}.tgz`;
+    if (packageObj.dependencies && name in packageObj.dependencies) {
+      packageObj.dependencies[name] = tarballUrl;
+    }
+    if (packageObj.devDependencies && name in packageObj.devDependencies) {
+      packageObj.devDependencies[name] = tarballUrl;
+    }
+  }
+  return packageObj;
```

Comment:
No need to return since its mutating the object (as the func name says)

Reply:
Whoops, good catch